src/Pages/Nodes.razor

```diff
@@ -511,4 +673,184 @@
         }
         return NodesColumnLayout.IsColumnVisible(column);
     }
+
+    // Liquidity Management Modal Methods
+    private async Task ShowLiquidityManagementModal(Node node)
+    {
+        _selectedNodeForLiquidity = node;
+        
+        // Convert sats to BTC for display
+        _minSwapSizeBtc = (decimal)new NBitcoin.Money(node.SwapMinAmountSats).ToUnit(NBitcoin.MoneyUnit.BTC);
+        _maxSwapSizeBtc = (decimal)new NBitcoin.Money(node.SwapMaxAmountSats).ToUnit(NBitcoin.MoneyUnit.BTC);
+        _balanceThresholdBtc = (decimal)new NBitcoin.Money(node.MinimumBalanceThresholdSats).ToUnit(NBitcoin.MoneyUnit.BTC);
+        _swapBudgetBtc = (decimal)new NBitcoin.Money(node.SwapBudgetSats).ToUnit(NBitcoin.MoneyUnit.BTC);
+        _budgetRefreshDays = node.SwapBudgetRefreshInterval.Days;
+        // Convert ratio from 0-1 to 0-100 for display
+        _maxSwapFeePercent = node.MaxSwapFeeRatio * 100m;
+        
+        if (_nodeLiquidityModal != null) await _nodeLiquidityModal.Show();
+    }
+
+    private async Task CloseLiquidityManagementModal()
+    {
+        if (_nodeLiquidityModal != null) await _nodeLiquidityModal.Close(CloseReason.UserClosing);
+        _selectedNodeForLiquidity = null;
+        await GetData();
+    }
+
+    private async Task SaveAndCloseLiquidityManagementModal()
+    {
+        if (_nodeLiquidityValidationsRef != null && await _nodeLiquidityValidationsRef.ValidateAll())
```

Comment:
nit: can the min-max validator be added here instead of showing a toast?

Reply:
Fixed, ValidateAll does the job and no longer a toast is required as the form is red and its not closing